src/pretix/base/services/orders.py

```diff
@@ -1563,7 +1563,8 @@ class OrderChangeManager:
     FeeValueOperation = namedtuple('FeeValueOperation', ('fee', 'value', 'price_diff'))
     AddFeeOperation = namedtuple('AddFeeOperation', ('fee', 'price_diff'))
     CancelFeeOperation = namedtuple('CancelFeeOperation', ('fee', 'price_diff'))
-    RegenerateSecretOperation = namedtuple('RegenerateSecretOperation', ('position',))
+    RegenerateSecretOperation = namedtuple('RegenerateSecretOperation', ('position', 'force_secret'))
```

Comment:
I think force_secret is a leftover from a previous attempt? It doesn't seem to be set anywhere and also breaks a test

Reply:
oops, yes
```suggestion
    RegenerateSecretOperation = namedtuple('RegenerateSecretOperation', ('position',))
```